docs/source/dashboard.rst

```diff
@@ -0,0 +1,130 @@
+shades have different meanings:
+    - Solid Blue: Managed memory under target (default 70% of memory)
+    - Solid Orange: Managed memory is close to the spilling target (default 70% of memory)
+
+For both blue and orange we have two more shades:
+    - Lighter shade: Unamanged memory
+    - Even lighter shade: Unmanaged recent memory. 
+
+    - Grey: Spilled to disk memory 
+
+For a detailed explanation on the different types of memory refer to the Worker Memory management
+documentation `Worker Memory management documentation <https://distributed.dask.org/en/latest/worker.html#memory-management>`_
+
+Processing/CPU/Occupancy
+------------------------
+
+**Task Processing** 
+
+.. figure:: images/dashboard_task_processing.png
+    :alt: NEEDS ALT TEXT
+
+The Processing plot, shows the tasks being processed across each worker. Ideally you want, each worker to 
+process an equally amount of tasks. If one of the bars is completely white it means that 
+worker is idle and waiting for tasks. This usually happens when the computations are close to finish (nothing 
+to worry about), but it can also mean that the distribution of the task across workers is not optimized and you
+will want to revisit your code. 
+
+In this plot you can also find the colors red and green: **(PENDING)**
+
+- Green : 
+- Red :
+
+In this plot on the dashboard we have two extra tabs with the following information:
+
+**CPU Utilization**
+
+CPU usage per-worker (this needs some love, haven't found a nice way of describing this) 
+
+**Occupancy**
+
+This tab shows the occupancy, in time, per worker. The total occupancy for a worker is the total expected runtime
+for all tasks currently on a worker. For example, an occupancy of 10s means (it's an estimation) that it will take the 
+worker 10s to execute all the tasks it has currently been assigned.
+
+Task Stream
+-----------
+
+The task stream is a view of all the tasks across threads. Each row represents a thread and each rectangle represent 
+an individual tasks. The color for each rectangle corresponds to the kind of task being performed and it matches the color 
+of the Progress plot (see Progress section). This means that all the individual tasks of kind `inc` for example will have 
+the same randomly assigned color. 
+
+There are certain colors that are reserved for a specific kinds of tasks:
+
+- Red: Transfer / communication 
+- Orange: Disk spilling
+- Black: Erred tasks
+- Grey: Serialization and deserialization
+- White: Idle thread
+
+.. figure:: images/dashboard_taskstream_healthy.png
+    :alt: NEEDS ALT TEXT
+
+.. figure:: images/dashboard_task_stream_unhealthy.png
+    :alt: NEEDS ALT TEXT
+
+In some scenarios the dashboard will have white spaces between each rectangles, this means that the during that time the thread
+is idle. Having to much white and red is an indication of not optimal use of resources, and you will want to revisit your 
+computations and/or the resources allocated. 
```

Comment:
Similar comment here about advising to revisit code without more specific feedback :smile:

Nits: `In some scenarios the dashboard` -> `In some scenarios, the dashboard`
`each rectangles, this means` -> `each rectangle. This means`
`Having to much` -> `Having too much` (how much is too much?)